include/dsAudio.h

```diff
@@ -286,7 +286,7 @@ dsError_t  dsSetAudioCompression(intptr_t handle, int compression);
  * This function returns the dialog enhancement level of the audio port corresponding to the specified port handle.
  *
  * @param[in] handle - Handle for the output audio port
- * @param[out] level - Pointer to Dialog Enhancement level (Value ranges from 0 to 16)
+ * @param[out] level - Pointer to Dialog Enhancement level (Value ranges from 0 to 12(as per 2.6 IDK) or 0 to 16(as per 2.4.1 IDK))
```

Comment:
Are we using only 2.6IDK and 2.4.1IDK? Do we need to generalise like any thing less than 2.6 should use 0 to 16 and 2.6 onwards it will use 0 to 12.

Reply:
The referce has been mentioned between 2.4 and 2.6. As per the discussion, the 2.5 should also have the same reuqirements.